substrate/docs/deprecation-process.md

```diff
@@ -0,0 +1,59 @@
+# Deprecation Process
+
+This deprecation process makes sense while we don’t [SemVer](https://semver.org/), after that this process will most likely change.
+
+As deprecation and removal of existing features can happen on any release we need to be mindful that external builders can be impacted by the changes we make.
+
+This process tries to mitigate this impact, while still keeping the devex as smooth as possible.
+
+First of all we need to create a new issue with the label `I11-deprecation` and add it to the [Runtime / FRAME](https://github.com/orgs/paritytech/projects/40) project. This will make sure that the issue is added to the [Deprecation board](https://github.com/orgs/paritytech/projects/40/views/12) for correct tracking.
```

Comment:
```suggestion
First of all we, need to create a new issue with the label `I11-deprecation` and add it to the [Runtime / FRAME](https://github.com/orgs/paritytech/projects/40) project. This will make sure that the issue is added to the [Deprecation board](https://github.com/orgs/paritytech/projects/40/views/12) for correct tracking.
```